src/library/starrating.h

```diff
@@ -22,11 +22,15 @@ class StarRating {
 
     static constexpr int kMinStarCount = 0;
 
+    static QSize getBaseSize() {
```

Comment:
From the function name it is not obvious what base size is. With all the HID scaling in mind this sounds form me as a normal and usable size. But it actually gives only the ratio between the width and length, normalized on a 1 pixel height widget. I have considered better names like `getMinimumSize()` or `getSizeForHeightOne()`, but then I realized that the scale factor is always the desired height and I have proposed to refactor it to `return getSizeForHight(int height);`
and rename `m_scaleFactor`  to `m_height`. 

If you like to keep the scale factor nomenclature, a better function name or/and explaining comment would be nice.

Reply:
Sorry for dropping the dsicussion. I considered the scale factor alternatives you proposed but none really fits IMHO. 'scale factor' is used only in this class so I'd like to keep that.

Regarding the get function, what about `getNormalizedSize()`? If you don't think that fits: for circles there's 'unit circle', so maybe we could call it `getUnitSize()`, though that might be understood as size of some self-contained element, not that of the entire stars container. Hence, I prefer `getNormalizedSize()`.
I'll also add a comment, of course.